src/main/java/io/debezium/connector/cassandra/transforms/CassandraTypeDeserializer.java

```diff
@@ -80,6 +81,7 @@ private CassandraTypeDeserializer() {
         tmp.put(DoubleType.class, new BasicTypeDeserializer(CassandraTypeKafkaSchemaBuilders.DOUBLE_TYPE));
         tmp.put(DecimalType.class, new BasicTypeDeserializer(CassandraTypeKafkaSchemaBuilders.DOUBLE_TYPE));
         tmp.put(Int32Type.class, new BasicTypeDeserializer(CassandraTypeKafkaSchemaBuilders.INT_TYPE));
+        tmp.put(IntegerType.class, new BasicTypeDeserializer(CassandraTypeKafkaSchemaBuilders.LONG_TYPE));
```

Comment:
This is likely borked - I don't think a `long` will be able to hold bigger `IntegerType` values.